migrations/versions/72912058602c_remove_classification_table_and_add_.py

```diff
@@ -0,0 +1,34 @@
+"""remove classification table and add file_name and classifiers into instrument
+
+Revision ID: 72912058602c
+Create Date: 2018-01-11 15:23:03.943466
+
+"""
+import sqlalchemy as sa
+
+from alembic import op
+from sqlalchemy.dialects.postgresql.json import JSONB
+from sqlalchemy.types import String
+
+# revision identifiers, used by Alembic.
+revision = '72912058602c'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('instrument',
+                  sa.Column('file_name', String(32)),
+                  schema='ras_ci'
+                  )
+    op.add_column('instrument',
+                  sa.Column('classifiers', JSONB()),
+                  schema='ras_ci'
+                  )
+    op.drop_table('classification',
+                  schema='ras_ci')
+
+
+def downgrade():
```

Comment:
Adding the downgrade script might be handy if things go wrong

Reply:
Me and Mark had a chat a few days ago on how we would tackle downgrades and we decided to stick to the standard ONS practise of snapshot rather then running a script in, so this becomes redundant